corehq/apps/change_feed/pillow.py

```diff
@@ -9,16 +9,18 @@
 from corehq.apps.change_feed.topics import get_topic
 from couchforms.models import all_known_formlike_doc_types
 import logging
+from pillowtop.checkpoints.manager import PillowCheckpoint, get_django_checkpoint_store
 from pillowtop.couchdb import CachedCouchDB
 from pillowtop.listener import PythonPillow
 
 
 class ChangeFeedPillow(PythonPillow):
 
-    def __init__(self, couch_db, kafka):
+    def __init__(self, couch_db, kafka, checkpoint):
         super(ChangeFeedPillow, self).__init__(couch_db=couch_db)
         self._kafka = kafka
         self._producer = KeyedProducer(self._kafka)
+        self._checkpoint = checkpoint
```

Comment:
it's sneakily overriding this: https://github.com/dimagi/pillowtop/pull/175/files#diff-6dac4be25b780fb79c406285662331fcR129 - can try to think through how to make more clear